### PR TITLE
Integrate Standard Guidelines into Client

### PR DESCRIPTION
**Pull request by Cwagne17**
PR completest the work on clark-client [Story 5026](https://app.clubhouse.io/clarkcan/story/5026/integrate-standard-guidelines-into-client). 

Changes standard-outcomes -> guidelines 
Implements new routes, and uses the seraching routes.

### CODE REVIEW

File: src/entity/guideline/error-messages.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export const GUIDELINE_ERRORS = {
+  INVALID_NAME: 'Name must be a defined non-empty string.',
+  ID_SET: 'Id has already been set and cannot be modified.',
+  INVALID_LEVELS: 'Level must be a valid level from the list',
+  INVALID_YEAR: 'Year must bea defined non-empty string.',
```
Comment (pzalep1) [inline src/entity/guideline/error-messages.ts:5]:
```suggestion
  INVALID_YEAR: 'Year must be a defined non-empty string.',
```

File: src/entity/guideline/guideline.ts [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+    }
+  }
+  /**
+   * Copies properties of outcome to this outcome if defined
+   *
+   * @private
+   * @param {Partial<Guideline>} outcome
+   * @memberof Guideline
+   */
+  private copyOutcome(outcome: any): void {
```
Comment (pzalep1) [inline src/entity/guideline/guideline.ts:132]: Could we change this to guideline to match the rest of the entity?

**Review by pzalep1 — CHANGES_REQUESTED** [review]
This looks great! Just two small changes.

**Review by EVogel1999 — APPROVED** [review]
Good job!  Looks great!  Thank you for renaming standard-outcomes to guidelines.

**Review by mfranz1 — APPROVED** [review]

**Review by jouypo23 — APPROVED** [review]

**Review by pzalep1 — APPROVED** [review]

**Pull request merged**